tests/integration/test_inspect_table.py

```diff
@@ -186,8 +185,6 @@ def test_inspect_entries(
 
                     assert df_lhs == df_rhs, f"Difference in data_file column {df_column}: {df_lhs} != {df_rhs}"
             elif column == 'readable_metrics':
-                right = right.asDict(recursive=True)
```

Comment:
dont need this anymore because of `spark.sql.execution.arrow.pyspark.enabled`, pandas DF turns tuple to dict

Reply:
Nice!